content/docs/1.4.0/deploy/install/_index.md

```diff
@@ -182,19 +182,30 @@ iscsi install successfully
 
 ### Installing NFSv4 client
 
-The command used to install a NFSv4 client differs depending on the Linux distribution.
+In Longhorn system, backup feature requires NFSv4, v4.1 or v4.2, and ReadWriteMany (RWX) volume feature requires NFSv4.1. Before installing NFSv4 client userspace daemon and utilities, make sure the client kernel support is enabled on each Longhorn node.
 
-For Debian and Ubuntu, use this command:
+- Check `NFSv4.1` support is enabled in kernel
+  ```
+  cat /boot/config-`uname -r`| grep CONFIG_NFS_V4_1
```

Comment:
We can enhance this check in the environment check script. This can be part of the `kernel functionality` check.

Reply:
Will add a check in the environment check script.